python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -297,6 +378,44 @@ async def telemetry_loop(self):
                 traceback=traceback.format_exc(),
             )
 
+    def convert_dimmMeasurement_data(self, data):
```

Comment:
so, this should probably be renamed `convert_dimm_measurement_data`. Mixing camelCase with snake_case is not a good idea.. 

The fact that the only class method you are calling here in `convert_float` (which I think don't belong in the class), tells me that this method should probably also be part of the same utils module as `convert_float` (`convert_to_float`) and `convert_int` (`convert_to_int`)..

Reply:
Done, though slightly different than you suggested because some values need to be converted to ints.